### ISSUE TITLE
Filter icon does not show up properly

### ISSUE THREAD

**Issue by stephenmoloney** [mirumee/saleor#2623]
### What I'm trying to achieve

To see the filter icon properly.

If I navigate to `http://localhost:8000/static/images/chevron-up.svg`
then I can see the big sized arrow ok.

### Steps to reproduce the problem
1. start docker-compose and run migrations, etc

### What I expected to happen

See filter down arrows ok.

### Screenshots

![image](https://user-images.githubusercontent.com/12668653/44293416-c3d89780-a282-11e8-99a7-a5b41acf4bc4.png)

**System information**
Operating system: Ubuntu 18.04
Browser: firefox
docker-compose: 1.22.0
docker: 18.06.0-ce

**Comment by stephenmoloney** [comment]
Seems like the down arrow is downloaded just fine in the networks tab

![image](https://user-images.githubusercontent.com/12668653/44297664-3e2e0980-a2cd-11e8-86f9-e58a462b3f14.png)

This warning in the console but it seems irrelevant (probably something to do with the server settings)

```text
Resource interpreted as Stylesheet but transferred with MIME type text/plain: "http://localhost:8000/static/assets/storefront.1fe699823524cd8f1f2f.css".
```

web logs on first page load with arrow

```
aleor-web | UWSGI uwsgi "GET /media/__sized__/products/saleor/static/placeholders/mugs/4_f39iMuK-crop-c0-5__0-5-255x255-70.jpg HTTP/1.1" 200 5685 11ms [PID:13:Worker-2] [RSS:91MB]
saleor-web | UWSGI uwsgi "GET /media/__sized__/products/saleor/static/placeholders/mugs/4_qFhBCFX-crop-c0-5__0-5-255x255-70.jpg HTTP/1.1" 200 5685 13ms [PID:15:Worker-4] [RSS:116MB]
saleor-web | UWSGI uwsgi "GET /media/__sized__/products/saleor/static/placeholders/mugs/box-01_zXmXsyF-crop-c0-5__0-5-255x255-70.jpg HTTP/1.1" 200 7065 11ms [PID:14:Worker-3] [RSS:115MB]
saleor-web | UWSGI uwsgi "GET /media/__sized__/products/saleor/static/placeholders/mugs/box-01_5Cn59PR-crop-c0-5__0-5-255x255-70.jpg HTTP/1.1" 200 7065 23ms [PID:12:Worker-1] [RSS:91MB]
saleor-web | UWSGI uwsgi "GET /media/__sized__/products/saleor/static/placeholders/mugs/4_jo74SUV-crop-c0-5__0-5-255x255-70.jpg HTTP/1.1" 200 5685 12ms [PID:13:Worker-2] [RSS:91MB]
saleor-web | UWSGI uwsgi "GET /media/__sized__/products/saleor/static/placeholders/mugs/4_8gQiAsD-crop-c0-5__0-5-255x255-70.jpg HTTP/1.1" 200 5685 13ms [PID:15:Worker-4] [RSS:116MB]
saleor-web | UWSGI uwsgi "GET /media/__sized__/products/saleor/static/placeholders/mugs/4_qFhBCFX-crop-c0-5__0-5-60x60-70.jpg HTTP/1.1" 200 1254 13ms [PID:14:Worker-3] [RSS:115MB]
saleor-web | UWSGI uwsgi "GET /static/images/arrow-up.svg HTTP/1.1" 200 867 0ms [PID:15:Worker-4] [RSS:116MB]
saleor-web | UWSGI uwsgi "GET /static/images/arrow-down.svg HTTP/1.1" 200 862 0ms [PID:15:Worker-4] [RSS:116MB]
saleor-web | UWSGI uwsgi "GET /static/images/arrow-up.svg HTTP/1.1" 200 867 0ms [PID:15:Worker-4] [RSS:116MB]
saleor-web | UWSGI uwsgi "GET /static/images/arrow-down.svg HTTP/1.1" 200 862 0ms [PID:15:Worker-4] [RSS:116MB]
```

Inspect the element 

![image](https://user-images.githubusercontent.com/12668653/44297691-b8f72480-a2cd-11e8-9338-ce9980f1565a.png)

**Comment by stephenmoloney** [comment]
It's because uwsgi isn't serving the mime types the way its needed. 

I can find a fix for this but something tells me that uwsgi is not an ideal way to serve the files in the first place but i could be wrong about that - i'm not a python developer.

**Comment by stephenmoloney** [comment]
Fix - #2627

**Comment by Pacu2** [comment]
I think this is already resolved
